Objects/scripts/XDSExpr.swift

```diff
@@ -966,52 +961,51 @@ indirect enum XDSExpr {
 			s += ["\n}\n"]
 			return s
 			
-		case .ifStatement(let es):
-			let parts = es.count
+		case .ifStatement(let condition, let exprs):
 			var lines = [String]()
-			
-			for i in 0 ..< parts {
-				let (condition, exprs) = es[i]
-				var keyword = "else"
-				if i == 0 {
-					keyword = "if"
-				}
-				
-				var conditionText = ""
-				switch condition {
-				case .jumpFalse(.loadImmediate(let c), _):
-					conditionText = XDSExpr.macroImmediate(c, .bool).forcedBracketed.text[0]
-				case .jumpFalse(let c, _):
-					conditionText = c.forcedBracketed.text[0]
-				case .jumpTrue(let c, _):
-					conditionText = "!" + c.forcedBracketed.text[0]
-				default:
-					// TODO: get rid of this as it messes with instruction count
-					// should always add condition as a jump
-					conditionText = condition.text[0]
-				}
-				if i == 0 {
-					lines += [keyword + " " + conditionText + " {\n"]
-				} else {
-					let last = lines.count - 1
-					lines[last] = lines[last].replacingOccurrences(of: "\n", with: "") + " " + keyword + " {\n"
+
+			// Pulled out in converting from a loop to a single condition + block
+			// Can reuse the below code when more complex if/else blocks are supported
```

Comment:
I think we can create a commented out ifElse case for use in future which contains the old code with the loop through each block. Then here we can clean this code up by removing the references to the loop counter since we know it will always be the 0 case for a singular if block.